packages/client/hooks/useUsageSnackNag.ts

```diff
@@ -60,14 +60,25 @@ const useUsageSnackNag = (insights: boolean) => {
           domains @include(if: $isUserNaggable) {
             tier
             suggestedTier
+            viewerOrganizations {
+              tierLimitExceededAt
+            }
           }
         }
       }
     `,
     {isUserNaggable}
   )
+
   const {domains} = viewer
-  const nag = domains?.find(({tier, suggestedTier}) => shouldNag(tier, suggestedTier))
+
+  // Do not use snack nag if the user has an organization with the limit exceeded, in this case we use a different warnings approach
+  const limitExceeded = domains?.find(({viewerOrganizations}) =>
```

Comment:
Fixed usage stats nagging snackbar. Now I don't show it in case limit exceeded flag is set.